client-side/css/styles.css

```diff
@@ -77,3 +77,25 @@ main {
   border: 2px solid green;
   height: 50vh;
 }
+
+/* •.¸¸¸.•*•.¸¸¸.•*•.¸¸¸.•*• */
+/* recipie display styles */
+/* •.¸¸¸.•*•.¸¸¸.•*•.¸¸¸.•*• */
+
+#container {
+  display: flex;
+  margin-left: 10%;
+  align-items: center;
+  justify-content: center; max-width: 70%;
+  margin: 10px;
+  border: 2px solid purple;
+}
+
+#imgDiv {
+  flex-basis: 40%;
+}
+
+#textDiv {
+  font-size: 22px;
+  padding-left: 20px;
+}
```

Comment:
given that ur JS code is trying to add these to the new elements as "classes" here:
```js
      main.classList.add('container')
      img.classList.add('imgDiv')
      textDiv.classList.add('textDiv')
```

they should be defined as classes in ur CSS, like `.container` && `.imgDiv` instead of defined as ids like `#container` && `#imgDiv`